iree/compiler/Dialect/HAL/Target/LLVM/LLVMTargetOptions.h

```diff
@@ -15,6 +15,7 @@
 #ifndef IREE_COMPILER_DIALECT_HAL_TARGET_LLVM_LLVMTARGETOPTIONS_H_
 #define IREE_COMPILER_DIALECT_HAL_TARGET_LLVM_LLVMTARGETOPTIONS_H_
 
+#include "iree/schemas/dylib_executable_def_builder.h"
```

Comment:
Then can I use the same style of enum for compile time and schema? Actually I declared Sanitizer enum in LLVMTargetOptions structure as well, but removed it because the schema has the same enum..

Reply:
@benvanik Do we want only one enum class in TargetOptions.h, and cast the enum to ubyte when generating an executable?
Or we want two enum classes, one in TargetOptions.h and another in flatbuffer schema? And we can switch-case to embed the information?

Reply:
Talked to Ben in chat, @inho9606 I think we should have an enum in the flatbuffer and an enum in target options - then just cast when building the flatbuffer in the compiler. Please also add a comment to state that the order in enum should be the same.

Reply:
Ah I see, I'll update it soon! THanks